R/GAMBLR.viz-package.R

```diff
@@ -17,6 +17,7 @@
 #' @importFrom GAMBLR get_ssm_by_sample
 #' @importFrom GAMBLR.helpers check_config_value
 #' @importFrom GAMBLR.helpers coding_vc
+#' @importFrom GAMBLR.helpers process_regions
```

Comment:
Why was this function added to this package?

Reply:
I forgot to mention before, but this function was imported because `heatmap_mutation_frequency_bin` uses it internally.
```
heatmap_mutation_frequency_bin: no visible global function definition
  for ‘process_regions’
```

Reply:
Ah, got it. Thanks for clarifying